docs/current/guides/snippets/argo-workflows/workflow.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: dagger-in-argo-
+spec:
```

Comment:
I would specify a namespace here.

Since we require Dagger to be installed via the K8s guide, I would be inclined to create this in the `dagger` namespace, as a lightweight check.